index.js

```diff
@@ -126,49 +122,57 @@ open.addEventListener('click', () => {
         </ul>
     </div>
     <div >
-    <img src="image/nature.png" alt="tonic-project" class="image"/>
+    <img src="${projects[position].images}" alt="tonic-project" class="image"/>
     </div>
     <div class="project-details-pop">
         <div class="about-project">
             <p>
-                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            ${projects[position].description}
             </p>
         </div>
         <div class="detailed-popup">
         <div class="technology">
             <ul>
               <li>
-                <p>html</p>
+                <p>${projects[position].language[0]}</p>
               </li>
                <li>
                  <p>
-                    css
+                 ${projects[position].language[1]}
                  </p>
                 </li>
                 <li>
                  <p>
-                    javascript
+                 ${projects[position].language[2]}
                  </p>
                </li>
             </ul>
          </div>
       <div class ="buttons">
-        <button type="submit" class="project1" >
-          See live<i class="fa fa-github"></i>
-        </button>
-        <button type="submit" class="project1" >
-          See source<i class="fa fa-github"></i>
-        </button>
+        <a href="${projects[position].liveLink}"><button type="submit" >
+        See Live<i class="fa fa-github"></i>
+        </button></a>
+        <a href="${projects[position].sourceLink}" target="_blank>
+        <button type="submit" class="source" >See source
+        <i class="fa fa-github"></i>
+        </button></a>
       </div>
     </div>
     </div>
   </div>`;
+  popup.id = 'project-pop';
+  works.appendChild(popup);
+  document.body.style.overflowY = 'hidden';
+}
 
-  main.appendChild(pop);
-  works.appendChild(main);
-  const closePopup = document.querySelector('.close');
-  closePopup.addEventListener('click', () => {
-    main.remove(pop);
+projectButtons.forEach((btn, index) => {
+  btn.addEventListener('click', () => {
+    addPopup(index);
```

Comment:
There are a few things that your popup is missing in order for it to behave the way it is expected as per project requirements.

1. Th close button is missing. kindly add a close button so that users can be able to close the popup when need be.

![image](https://user-images.githubusercontent.com/38088818/174439741-7c6fe018-7490-4b71-97f6-88a517dbae9e.png)

2. The background with the portfolio content must be blurry when the popup is active so that it stands out from the main page.

![image](https://user-images.githubusercontent.com/38088818/174439867-a1f607ac-2b64-4020-8973-74643b5cc1d2.png)

FIGMA:
![image](https://user-images.githubusercontent.com/38088818/174439962-de534b19-6a74-48e3-a034-dc37e8717bee.png)